en/docs/assets/css/theme.css

```diff
@@ -215,3 +215,118 @@
     transform: scale(1);
     transition-timing-function: cubic-bezier(0.680, 0, 0.265, 1);
 }
+
+.feedback {
+    background-color: #555555;
+    border-radius: 0px;
+    color: #ffffff;
+    display: inline-block;
+    position:fixed;
+    font-size: 0.9em;
+    font-weight: 700;
+    margin-top: 0px;
+    margin-right: 0px;
+    margin-bottom: 1em;
+    margin-left: 0px;
+    padding-top: 2px;
+    padding-right: 8px;
+    padding-bottom: 2px;
+    padding-left: 8px;
+    text-transform: uppercase;
+    top: 68% !important;
+    transform: rotate(90deg);
+    transform-origin: top left;
+    left: 100%;
+}
+
+.header {
+    background-color: rgb(96, 125, 139);
+    box-sizing: border-box;
+    color: white;
+    position: relative;
+}
+
+.h1 {
```

Comment:
Why do we have `.h1` class name when we have html element `h1`. Please use meaningful class names